pybedtools/bedtool.py

```diff
@@ -584,19 +593,23 @@ def __init__(self, fn=None, from_string=False, remote=False):
         _tags[tag] = self
         self._hascounts = False
         self._file_type = None
+        self.seqfn = None
```

Comment:
this is another change I made not related exclusively with type annotation... but I see there are several method that add an attribute to the Bedtool object when run... for example the sequence method.

IMHO It makes more sense to just have these attributes by initalised with the bedtool object as None values and then assign a value to them when the specific method is run... 

other method that require the attribute to be present (like print_squence) then just check if the attribute is not None